src/scripts/concatenate.mjs

```diff
@@ -90,6 +90,20 @@ function removeFrontMatter(mdText) {
   }
 }
 
+// Unified tools put escapes in places
+// We could get around this by forking mdast-util-to-string as described here:
+// https://github.com/remarkjs/strip-markdown/issues/28#issuecomment-1290847745
+function cleanUpEscapes(text) {
+  return text
+    // Fix stripped markdown escaping such as `_` to `\_`
+    .replaceAll('\\\_', '_')
+    .replaceAll('\\<', '<')
+    .replaceAll('\\[', '[')
+    .replaceAll(/^\\-/gm, '-')
+    .replaceAll('\\.', '.')
```

Comment:
Just to check: I presume this isn't a regexp, otherwise the dot would swallow any character, right?

Reply:
Yes, because it's a string instead of between slashes like the previous line, Node treats it like a string value to replace and not a regexp. I've checked the output.